command/ca/token.go

```diff
@@ -513,20 +526,29 @@ func provisionerPrompt(ctx *cli.Context, provisioners provisioner.List) (provisi
 	}
 
 	if len(provisioners) == 1 {
-		var id, name string
+		var name, value string
 		switch p := provisioners[0].(type) {
```

Comment:
Maybe use `switch p.GetType()` as you did above.

Reply:
We want to use p as type, for example, to get struct properties or call methods that are not part of the provisioner interface.

Reply:
gotcha 👍